test_data/yast/encryption/activate_encrypted_volume.yaml

```diff
@@ -6,3 +6,9 @@ device_status:
     cipher: 'aes-xts-plain64'
     key_location: 'dm-crypt'
     mode: 'read/write'
+partitioning_deletion_entries:
```

Comment:
let's delete this file then.

Reply:
still used by schedule/yast/encryption/activate_encrypted_volume.yaml  so we can keep it for now